index.html

```diff
@@ -2,64 +2,1083 @@
+                        <a href="#">Justice Ain't Much If You Can't Afford It. A Civil Law Overhaul Is Overdue</a>
+                        <div class="byline">By <a href="#">MARTHA BERGMARK</a></div>
+                      </li>
+                      <li class="groupItem">
+                          <img src="./assets/images/mathaleft.png" alt="">
+                          <a href="#">Justice Ain't Much If You Can't Afford It. A Civil Law Overhaul Is Overdue</a>
+                          <div class="byline">By <a href="#">MARTHA BERGMARK</a></div>
+                      </li>
+
+                      <li class="groupItem">
+                          <img src="./assets/images/mathaleft.png" alt="">
+                          <a href="#">Justice Ain't Much If You Can't Afford It. A Civil Law Overhaul Is Overdue</a>
+                          <div class="byline">By <a href="#">MARTHA BERGMARK</a></div>
+                      </li>
+          
+                      <li class="groupItem">
+                        <img src="./assets/images/mathaleft.png" alt="">
+                        <a href="#">What Happened Before the Big Bang?</a>
+                        <div class="byline">By <a href="#">MARTHA BERGMARK</a></div>
+                      </li>
+                    </ul>
+                  </section>
+                
+
+                <!-- %%%%%%%%%%%%%%% LatestNews Section %%%%%%%%%%% -->
+                </section class="latest-news-section">
+                  <div id="sticky-wrapper">
+                      <h2>Latest News</h2>
+                      
+                      <ul class="list-group latest-news">
+                        <article>
+                          <li class="groupItem">
+                            <img src="./assets/images/smallimage1.jpg"  alt="">
+                            <a id="title" href="#">'RHOBH' Season 9 Cast Choose Between Kyle and Lisa</a>
+                          </li>
+                      </article>
+
+                      <article>
+                          <li class="groupItem">
+                            <img src="./assets/images/smallimage1.jpg"  alt="">
+                            <a id="title" href="#">'RHOBH' Season 9 Cast Choose Between Kyle and Lisa</a>
+                          </li>
+                      </article>
+
+                      <article>
+                          <li class="groupItem">
+                            <img src="./assets/images/smallimage1.jpg"  alt="">
+                            <a id="title" href="#">'RHOBH' Season 9 Cast Choose Between Kyle and Lisa</a>
+                          </li>
+                      </article>
+
+                      <article>
+                          <li class="groupItem">
+                            <img src="./assets/images/smallimage1.jpg"  alt="">
+                            <a id="title" href="#">'RHOBH' Season 9 Cast Choose Between Kyle and Lisa</a>
```

Comment:
You seem to be using multiple the same `id` in multiple places. Consider making these `id` unique across other elements.